packages/web3-shared/base/src/utils/resolver.ts

```diff
@@ -215,7 +215,7 @@ export const resolveNextIdWalletName = createLookupTableResolver<NetworkPluginID
     },
 )
 
-export const resolveNextIdPlatformPluginId = createLookupTableResolver<NextIDPlatform, NetworkPluginID | undefined>(
+export const resolveNetworkPluginID = createLookupTableResolver<NextIDPlatform, NetworkPluginID | undefined>(
```

Comment:
I don't think the name is very reminiscent of NextID

Reply:
DONE